lib/twitter_cldr/js/mustache/data_bundle.coffee

```diff
@@ -0,0 +1,38 @@
+###
+// Copyright 2012 Twitter, Inc
+// http://www.apache.org/licenses/LICENSE-2.0
+// TwitterCLDR (JavaScript) v{{version}}
+// Authors:     Cameron Dutro [@camertron]
+                Kirill Lashuk [@KL_7]
+                portions by Sven Fuchs [@svenfuchs]
+// Homepage:    https://twitter.com
+// Description: Provides date, time, number, and list formatting functionality for various Twitter-supported locales in Javascript.
+###
+
+{{#source_map}}
+###
+//@ sourceMappingURL={{locale}}.map
+###
+{{/source_map}}
+
+# required alias definition that adds twitter_cldr to Twitter's static build process
+###<<module_def>>###
+
+
+TwitterCldrDataBundle =
+{{{contents}}}
+
+root = if exports?
+  exports
+else
+  this.TwitterCldrDataBundle = {}
+  this.TwitterCldrDataBundle
+
+root[key] = obj for key, obj of TwitterCldrDataBundle
```

Comment:
(Still lurking :smile:) From what I understand, the Coffeescript compiler automatically wraps all the twitter-cldr-js code in a closure. Because of this, `TwitterCldrDataBundle` and `TwitterCldr` aren't truly global variables - they're defined in the context of that outermost closure. Lines 25-31 exist to support AMD/requirejs/etc as well as the browser - all the twitter-cldr-js methods and properties need to get copied over to `exports` (or `this` in the case of the browser). I understand that `this` is equal to `window` in the browser... which is a problem I don't know how to solve.